config/crd/bases/weave.works_profiledefinitions.yaml

```diff
@@ -46,6 +46,11 @@ spec:
                       description: Chart defines properties to access a remote chart.
                         This is an optional value. It is ignored in case Path is defined.
                       properties:
+                        defaultValues:
+                          description: DefaultValues holds the default values for
+                            this Helm release Artifact. These can be overridden by
+                            the user, but will otherwise apply.
+                          type: string
```

Comment:
string? I was expecting a map[string]interface or something. What does it look like to configure this field?

Reply:
did you even read the description tho?

Reply:
NO THATS FOR LOSERS